scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -419,7 +419,7 @@ abstract class FixedPathSource(path: String*) extends FileSource {
   def localPaths = path.toList
   def hdfsPaths = path.toList
 
-  override def toString = getClass.getName + path
+  override def toString = getClass.getName + path.mkString(",")
```

Comment:
The test you added passes for me on develop. Is there a better one that currently fails on develop and this branch fixes it? That might help clarify what this fix does (I don't quite understand it either).